tests/unit/type/hash.tcl

```diff
@@ -650,27 +650,27 @@ start_server {tags {"hash"}} {
 
     test {HINCRBYFLOAT over hash-max-listpack-value encoded with a listpack} {
         set original_max_value [lindex [r config get hash-max-ziplist-value] 1]
-        r config set hash-max-ziplist-value 8
+        r config set hash-max-listpack-value 8
         
-        # hash's value exceeds hash-max-ziplist-value
+        # hash's value exceeds hash-max-listpack-value
         r del smallhash
         r del bighash
         r hset smallhash tmp 0
         r hset bighash tmp 0
-        r hincrbyfloat smallhash tmp 12345678
-        r hincrbyfloat bighash tmp 123456789
+        r hincrbyfloat smallhash tmp 0.000005
+        r hincrbyfloat bighash tmp 0.0000005
         assert_encoding listpack smallhash
         assert_encoding hashtable bighash
 
-        # hash's field exceeds hash-max-ziplist-value
+        # hash's field exceeds hash-max-listpack-value
         r del smallhash
         r del bighash
         r hincrbyfloat smallhash 12345678 1
         r hincrbyfloat bighash 123456789 1
```

Comment:
i suppose you should change these too?

Reply:
Sorry, I don't quite understand what you mean. 
Do you mean to use fractional numbers here too? 
Here is test for field over hash-listpack-max-value, 
do you want field to use fractional numbers? or value?

Reply:
well, you're right that we're not incrementing the field name (we're incrementing the value).
but from the perspective of listpack it doesn't matter.
i.e. listpack can save strings or integers (can't save doubles), so by using a string that's an integer value, you're not really saving a long string.
so i guess in this case you should use `abcdefg` or `xxxxxxx` rather than number (for the value you couldn't have done that since it must be a number, which is why we used fractional numbers, to force it to save string rather than integer).

we probably shouldn't be looking into this that much, but since we did.. let's do that.

Reply:
Okay, it's been fixed again, and I think this should be the last time.